source/configuration_manual/authentication/oauth2.rst

```diff
@@ -253,60 +266,88 @@ Oauth2 overrides some of the default HTTP client and SSL settings:
 You can override these and any other HTTP client or SSL settings by placing
 them inside :dovecot_core:ref:`oauth2` named filter.
 
-.. code-block:: none
 
-  ### OAuth2 password database configuration
+.. dovecot_core:setting:: oauth2_tokeninfo_url
+   :values: @string
+
+   URL for verifying token validity. Token is appended to the URL. Example:
+   ``oauth2_tokeninfo_url = http://endpoint/oauth/tokeninfo?access_token=``
+
+.. dovecot_core:setting:: oauth2_introspection_url
+   :values: auth, get, post, local
```

Comment:
:default: missing

Reply:
This isn't true now. The default is empty, and an empty value is allowed in values. Should also say that this must be set non-empty for oauth2 to be enabled.

Reply:
unchanged